pkg/sensors/tracing/generickprobe.go

```diff
@@ -528,7 +529,12 @@ func createGenericKprobeSensor(
 		kprobes[i].Syscall = syscall
 
 		for idx := range syms {
-			out, err := addKprobe(syms[idx], &kprobes[i], &in)
+			// if we use kprobe-multi, we need to share the maps.
+			// Otherwise, we create new ones.
+			if !useMulti {
```

Comment:
is this necessary? there's check in NewKernelSelectorState that ensures we get new maps object if it's nil,
maybe we could just pass global selMaps for useMulti and nil otherwise?

Reply:
What "global selMaps" are you referring to? In the last version, I removed the global variable based on @kkourt review.

Reply:
Updated the PR based on your comment. Sorry, I initially got a bit confused.